code/modules/holiday/spacemas.dm

```diff
@@ -488,6 +488,13 @@ var/static/list/santa_snacks = list(/obj/item/reagent_containers/food/drinks/egg
 		else
 			src.UpdateOverlays(null, "fire")
 
+	ephemeral //Disappears except on xmas
```

Comment:
Can we get these added to the mapping guidelines maybe?

Reply:
Added in f67c875